server/service/http/api.go

```diff
@@ -150,15 +150,16 @@ func (a *API) getShardTables(writer http.ResponseWriter, req *http.Request) {
 	var getShardTables GetShardTables
 	err := json.NewDecoder(req.Body).Decode(&getShardTables)
 	if err != nil {
-		a.respondError(writer, ErrParseRequest, "decode request body failed")
+		log.Error("decode request body failed", zap.Error(err))
+		a.respondError(writer, ErrParseRequest, fmt.Sprintf("decode request body failed, cause: %v", err))
 		return
 	}
 	log.Info("get shard tables request", zap.String("request", fmt.Sprintf("%+v", getShardTables)))
 
 	c, err := a.clusterManager.GetCluster(req.Context(), getShardTables.ClusterName)
 	if err != nil {
 		log.Error("get cluster failed", zap.String("clusterName", getShardTables.ClusterName), zap.Error(err))
-		a.respondError(writer, ErrGetCluster, fmt.Sprintf("get cluster failed, clusterName:%s", getShardTables.ClusterName))
+		a.respondError(writer, ErrGetCluster, fmt.Sprintf("get cluster failed, clusterName: %s, cause: %v", getShardTables.ClusterName, err))
```

Comment:
```suggestion
		a.respondError(writer, ErrGetCluster, fmt.Sprintf("get cluster failed, clusterName: %s, cause: %s", getShardTables.ClusterName, err.Error()))
```

Reply:
We do not need to exposed error stack to `CeresDB`, just throw error message is enough.